osgp/platform/osgp-adapter-ws-admin/src/main/java/org/opensmartgridplatform/adapter/ws/admin/application/services/DeviceLogItemSpecifications.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.admin.application.services;
+
+import static org.opensmartgridplatform.shared.utils.WildcardUtil.replaceWildcards;
+
+import java.util.Date;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+
+import org.opensmartgridplatform.logging.domain.entities.DeviceLogItem;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.jpa.domain.Specification;
+
+public final class DeviceLogItemSpecifications {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DeviceLogItemSpecifications.class);
+
+    private static final String DEVICE_IDENTIFICATION = "deviceIdentification";
+    private static final String ORGANISATION_IDENTIFICATION = "organisationIdentification";
+    private static final String MODIFICATION_TIME = "modificationTime";
+
+    private DeviceLogItemSpecifications() {
+        // Prevents creation of DeviceLogItemSpecifications objects
+    }
+
+    public static final Specification<DeviceLogItem> NONE = (final Root<DeviceLogItem> r, final CriteriaQuery<?> q,
+            final CriteriaBuilder cb) -> cb.or();
+
+    public static final Specification<DeviceLogItem> ALL = (final Root<DeviceLogItem> r, final CriteriaQuery<?> q,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1213)